apps/condo/domains/organization/integrations/sbbol/SbbolRequestApi.js

```diff
@@ -25,18 +26,19 @@ const REFRESH_TOKEN_TTL = 30 * 24 * 60 * 60 // its real TTL is 180 days bit we n
  */
 
 class SbbolRequestApi {
-    constructor (accessToken) {
-        // TODO(pahaz): use protected_port?
-        const { protected_host: hostname, port } = SBBOL_CONFIG
```

Comment:
protected_host can have an own protected_port.

Reply:
Now it does not have, if it will, I will add it to config.